includes/class-wc-gateway-ppec-client.php

```diff
@@ -547,7 +547,7 @@ protected function get_details( $details, $discounts, $rounded_total, $total ) {
 				// Add line item to make up different between WooCommerce
 				// calculations and PayPal calculations.
 				$diff = round( $details['total_item_amount'] - $rounded_total, $decimals );
-				if ( $diff != 0 ) {
+				if ( abs( $diff ) < 0.0001 ) {
```

Comment:
It uses the formula `|a - b| < e` to see how close a and b are for a small e (the smaller the better). In this case b = 0 and e = 0.0001.

As far as I checked debug logs we were sending PP a very small number which was caused by the calculation above on floats, so PP didn't like that number.